dts/arm/st/h7rs/stm32h7rs.dtsi

```diff
@@ -48,9 +48,9 @@
 		};
 	};
 
-	/* System data RAM accessible over AXI bus: AXI SRAM1 in CD domain */
+	/* System data RAM accessible over AXI bus: AXI SRAM1 to AXI SRAM4 in CD domain */
```

Comment:
Doing so, should we also remove the `sram1`, `sram2`, `sram3` nodes that now overlap this one?